client/my-sites/upgrades/components/domain-warnings/pending-gapps-tos-notice.jsx

```diff
@@ -42,7 +44,9 @@ const PendingGappsTosNotice = React.createClass( {
 	},
 
 	getGappsLoginUrl( email, domain ) {
-		return `https://accounts.google.com/AccountChooser?Email=${ email }&service=CPanel&continue=https%3A%2F%2Fadmin.google.com%2F${ domain }%2FAcceptTermsOfService%3Fcontinue%3Dhttps%3A%2F%2Fmail.google.com%2Fmail%2Fu%2F1`;
+		return `https://accounts.google.com/AccountChooser?Email=${ email }&service=CPanel' +
```

Comment:
Looks like there's a mismatch in punctuation here with ` being closed with '.
The output of this function looks like this (email: XX, domain: YY):

<img width="763" alt="screen shot 2016-12-19 at 00 22 08" src="https://cloud.githubusercontent.com/assets/4335450/21293757/74a36666-c581-11e6-979b-649d23ce8575.png">

This should be a quick fix with just changing those `'`s :)